src/librbd/ImageCtx.h

```diff
@@ -154,6 +192,9 @@ namespace librbd {
     xlist<operation::ResizeRequest<ImageCtx>*> resize_reqs;
 
     io::ImageRequestWQ *io_work_queue;
+    ImageQosStatusSingleton* qos_status;
+    SafeTimer *m_timer;
```

Comment:
Please don't introduce a new timer for such a minor purpose -- we are trying to reduce our thread usage.